tests/goldens/config/base/publish.yml

```diff
@@ -0,0 +1,4 @@
+repository: https://gitlab.com/getindata/dataops/some_repo.git
```

Comment:
Wouldn't we like to test if the file is parsed and used correctly? 'Cause right now I do not see any test checking particularly this.

Reply:
What files? Can you propose some UC for testing?

Reply:
I mean `publish.yml`

Right now you extract `repository`, `username` and `branch` out of this file. But it is not tested if they are properly passed to the Git implementation or sth like that. But well, maybe it is not a valuable test, as there's no processing of this data, just passing them "raw" as arguments to `Repo`'s methods?